### ISSUE TITLE
doesn't respect frozen properties of the primary realm

### ISSUE THREAD

**Issue by jdalton** [caridy/secure-javascript-environment#15]
In the primary realm:
```js
document.bar = { a:1, b: 2 }
Object.freeze(document.bar)
Object.isFrozen(document.bar) // => true
```
Within the shadow realm:
```js
document.bar.c = 3
document.bar // => { a:1, b:2, c:3 }
Object.isFrozen(document.bar) // => false
```

**Comment by jdalton** [comment]
In the case for this repro I used the webcomponent example and before clicking the "evaluate in new sandbox" I ran the code in the primary realm. Then in the sandbox I just had it do "debugger" and ran the test snippet in the shadow realm via the dev console.

**Comment by jdalton** [comment]
Closed by #16.
